### ISSUE TITLE
CNCF Conformance tests for CAPI Clusters

### ISSUE THREAD

**Issue by gawertm** [giantswarm/roadmap#1486]

**Comment by gawertm** [comment]
@puja108 I think this is also something to be tracked in Horizon for now and potentially go into a new team

**Comment by gawertm** [comment]
@puja108 @teemow is it ok to track further in Horizon (maybe until new releng team is in place)?
see also my comment [here](https://github.com/giantswarm/roadmap/issues/685#issuecomment-1308822018) about what we see as in scope for Rocket and what would be follow-ups in potentially another team

**Comment by JosephSalisbury** [comment]
@Rotfuks we were trying to find a home for this issue in horizon planning - do you feel that turtles owns conformance in general? would you dispatch this issue?

**Comment by JosephSalisbury** [comment]
yeah, maybe tinkerers is somehow easier - @puja108 what do you think (being as we just talked about this)

**Comment by puja108** [comment]
Fine by me, as long as there's some defined process in place to make sure that in the end we are conform and ownership is clearly handled along this process. If Tinkerers does this manually or automates the dispatch, both work for me.

**Comment by JosephSalisbury** [comment]
paging dr @yulianedyalkova for input when she's back from vacation

**Comment by Rotfuks** [comment]
Yeah, i'd say it fits to Turtles Vision of a good kubernetes foundation - that this foundation is cncf conform...I'll pull it in my backlog.

**Comment by yulianedyalkova** [comment]
I agree that pipelines and tests should belong to Turtles once we are done with the initial setup. When it comes to test failures, we will probably directly ping provider teams to look into them as we do with the other types too.

**Comment by JosephSalisbury** [comment]
@yulianedyalkova i don't follow there - pipelines and tests should belong to turtles, but tinkerers would take care of pinging?

**Comment by JosephSalisbury** [comment]
gotcha - okay, so turtles owns it, pings happen. works for me

where do we document ownership nowadays? @puja108 do you have a place?

**Comment by puja108** [comment]
main source of truth is `g/github` these days, not sure if this can be pinned down to a github repo here. Also, for this kind of "complex" relationship, I feel some intranet page might make sense, to explain how the ownership for test infra / pipelines / tests / fixing generally flows. From what I'm reading it is similar across types, so maybe we can document it in general, and then just mentioning the types for an overview. Yulia, could Tinkerers document that? It feels you have a good mental picture of it, which we could put into a real picture/diagram or so.

**Comment by yulianedyalkova** [comment]
I don't know if the relationship is complex enough for a whole doc, the desired state is "teams own their pipelines and tests, Tinkerers provide the infrastructure (including common building blocks like alerting)". But we're not there yet and we can try to keep the current state in a doc if that would be useful for teams.
